<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>865</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>60</y>
      <width>80</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Agregar</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>110</y>
      <width>80</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Eliminar</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_3">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>160</y>
      <width>80</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Informacion</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_4">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>220</y>
      <width>80</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Salir</string>
    </property>
   </widget>
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>50</y>
      <width>621</width>
      <height>421</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="page">
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>30</y>
        <width>101</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Agregar producto</string>
      </property>
     </widget>
     <widget class="QStackedWidget" name="stackedWidget_2">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>230</y>
        <width>281</width>
        <height>61</height>
       </rect>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="page_3">
       <widget class="QLabel" name="label_6">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>20</y>
          <width>61</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Cantidad:</string>
        </property>
       </widget>
       <widget class="QSpinBox" name="cantidadProd">
        <property name="geometry">
         <rect>
          <x>100</x>
          <y>20</y>
          <width>71</width>
          <height>25</height>
         </rect>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="page_4">
       <widget class="QLabel" name="label_7">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>20</y>
          <width>61</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>Descuento:</string>
        </property>
       </widget>
       <widget class="QDoubleSpinBox" name="descuentoProd">
        <property name="geometry">
         <rect>
          <x>100</x>
          <y>20</y>
          <width>62</width>
          <height>25</height>
         </rect>
        </property>
       </widget>
      </widget>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>200</y>
        <width>81</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Tipo producto:</string>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBox">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>200</y>
        <width>151</width>
        <height>24</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>Producto con stock</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Producto con descuento</string>
       </property>
      </item>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>70</y>
        <width>121</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Nombre del producto</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>110</y>
        <width>49</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Precio:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>148</x>
        <y>150</y>
        <width>71</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Descripcion:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="nombreProd">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>70</y>
        <width>121</width>
        <height>24</height>
       </rect>
      </property>
     </widget>
     <widget class="QDoubleSpinBox" name="precioProd">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>110</y>
        <width>101</width>
        <height>25</height>
       </rect>
      </property>
     </widget>
     <widget class="QTextEdit" name="descripcionProd">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>150</y>
        <width>171</width>
        <height>41</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_5">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>310</y>
        <width>80</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>Agregar</string>
      </property>
     </widget>
     <widget class="QLabel" name="statusAgregar">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>350</y>
        <width>171</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Se ha agregado un producto.</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_2">
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>40</y>
        <width>101</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Eliiminar producto</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_9">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>120</y>
        <width>121</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Nombre del producto:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="eliminarProd">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>120</y>
        <width>113</width>
        <height>24</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_6">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>180</y>
        <width>80</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>Eliminar</string>
      </property>
     </widget>
     <widget class="QLabel" name="statusEliminar">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>250</y>
        <width>151</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Se ha eliminado el producto</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_5">
     <widget class="QLabel" name="label_10">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>50</y>
        <width>111</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Mostrar informacion</string>
      </property>
     </widget>
     <widget class="QTextEdit" name="mostrarInformacion">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>90</y>
        <width>371</width>
        <height>251</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_11">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>360</y>
        <width>61</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Valor total:</string>
      </property>
     </widget>
     <widget class="QLabel" name="valorTotal">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>360</y>
        <width>49</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>865</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
